regolith/storage.py

```diff
@@ -88,18 +88,19 @@ def push_git(store, path, rc):
     subprocess.check_call(cmd, cwd=storedir)
     if "default_user_id" in rc:
         cmd = ["git", "commit", "-m", f"{rc.default_user_id} regolith auto-store commit"]
```

Comment:
see my comment on the other PR.  It looks as we may need ``default_github_id`` in regolithrc.json